pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -21,50 +36,491 @@ func init() {
 	defaultEnforcer = regoEnforcer
 }
 
+//go:embed framework.rego
+var FrameworkCode string
+
+//go:embed policy.rego
+var PolicyCode string
+
+var Indent string = "    "
+
 // RegoEnforcer is a stub implementation of a security policy, which will be
 // based on [Rego] policy language. The detailed implementation will be
 // introduced in the subsequent PRs and documentation updated accordingly.
 //
 // [Rego]: https://www.openpolicyagent.org/docs/latest/policy-language/
-type RegoEnforcer struct{}
+type RegoEnforcer struct {
+	// Rego which describes policy behavior (see above)
+	behavior string
```

Comment:
It looks like we never actually use this field, other than in the same function that initializes the `RegoEnforcer`. Can we remove it?

Reply:
@matajoh thoughts? This is an area I'm not very familiar with.

Reply:
At the moment, `behavior` is the same for all policies (because it is loaded from `PolicyCode`) but once we accept Rego this will be authored by customers and potentially distinct. As such, I want to record it in the Enforcer for debugging (and/or error logging) purposes. We can add it then, but it seemed a good idea to add it now in anticipation.

Reply:
That seems fine to me.